pkg/productinfo/types.go

```diff
@@ -133,6 +131,12 @@ type NetworkPerfMapper interface {
 	MapNetworkPerf(vm VmInfo) (string, error)
 }
 
+// Cache interface collects cache operations
+type Cache interface {
```

Comment:
let's find a better name for this, cache is too generic and we don't implement one; rather one of our components is "backed" by a cache.
Suggestions: ProductStorer, DetailsStorer